src/EA.Weee.Core/Organisations/Base/OrganisationViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace EA.Weee.Core.Organisations.Base
+{
+    using System.ComponentModel;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.DataStandards;
+
+    public abstract class OrganisationViewModel
```

Comment:
@stevelee-civica Notice how the models have been moved to EA.Weee.Core project. This is so that they can be used by the OrganisationTransactionData and during the request / request handler process.